pyodide_build/config.py

```diff
@@ -66,6 +66,7 @@ def _get_make_environment_vars(self) -> Mapping[str, str]:
             capture_output=True,
             text=True,
             env={"PYODIDE_ROOT": str(self.pyodide_root)},
+            check=False,
```

Comment:
I'm curious why we need to set it to `False`, could we switch it to `True` instead?

Reply:
```suggestion
            check=True,
```

Reply:
Yeah this explicit `check=False` is a really nice rule.